MimeUtility.cs

```diff
@@ -16,11 +16,11 @@ public static class MimeUtility
         /// </summary>
         public const string UnknownMimeType = "application/octet-stream";
 
-        static Lazy<ReadOnlyDictionary<string, string>> _lazyDictExtensions = new Lazy<ReadOnlyDictionary<string, string>>(
+        static readonly Lazy<ReadOnlyDictionary<string, string>> _lazyDictExtensions = new Lazy<ReadOnlyDictionary<string, string>>(
```

Comment:
please do the changes inside this file in a separate PR. it's release related.

Reply:
Thank you for your review and feedback!

This fixes IDE0044 "Add readonly modifier". It's more code style related and non-braking.
I thought it wasn't worth a separate PR. But OK, I will move it into separate PR.